pkg/authenticator/helmsecret.go

```diff
@@ -0,0 +1,33 @@
+package authenticator
+
+import (
+	"os"
+)
+
+type helmsecret struct {
+}
+
+var _ Authenticator = (*helmsecret)(nil)
+
+func NewHelmSecret() *helmsecret {
+	return &helmsecret{}
+}
+
+func (s *helmsecret) GetAuthFileName() (string, error) {
```

Comment:
```suggestion
func (s *helmsecret) AuthFilename() (string, error) {
```
If you want to match a previous nit.